Source/Upstream: Yes, in v238-stable branch

From d555d4cf8e1cf1d065e49e6eba88e7f8c780449e Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Wed, 14 Mar 2018 06:36:29 +1000
Subject: [PATCH 19/51] hwdb: add axis override for the Razer Blade Stealth
 (#8436)

This touchpad has heavy jitter, set a fuzz of 8 to work around this.

From https://bugs.freedesktop.org/show_bug.cgi?id=105409

(cherry picked from commit 539ad37f18543d436ef9839e6bec20319ada53a2)
---
 hwdb/60-evdev.hwdb | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/hwdb/60-evdev.hwdb b/hwdb/60-evdev.hwdb
index f56be3687..7e4ae61ea 100644
--- a/hwdb/60-evdev.hwdb
+++ b/hwdb/60-evdev.hwdb
@@ -410,6 +410,17 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pn*02173BG*:*pvrThinkPadEd
  EVDEV_ABS_35=916:6077:55
  EVDEV_ABS_36=653:5395:116
 
+#########################################
+# Razer
+#########################################
+
+# Razer Blade Stealth
+evdev:name:1A586753:00 06CB:8323 Touchpad:dmi:*svnRazer:pnBladeStealth:*
+ EVDEV_ABS_00=::12:8
+ EVDEV_ABS_01=::11:8
+ EVDEV_ABS_35=::12:8
+ EVDEV_ABS_36=::11:8
+
 #########################################
 # Samsung
 #########################################
-- 
2.16.3

